### PR TITLE
Added expense description under title in TransactionCard.kt

### LINKED ISSUE #684: Show Description In Expense List

### PR DESCRIPTION
**Pull request by mhss1**
## Pull Request (PR) Checklist
Please check if your pull request fulfills the following requirements:
- [x] The PR is submitted to the `develop` branch.
- [x] I've read the **[Contribution Guidelines](https://github.com/ILIYANGERMANOV/ivy-wallet/blob/main/CONTRIBUTING.md)**.
- [x] The code builds and is tested on an actual Android device.
- [x] I confirm that I've run the code locally and everything works as expected.
- [ ] I confirm that I've run Ivy Wallet's UI tests (`androidTest`) and all tests are passing
  successfully.

_Important: Don't worry if you experience flaky UI tests. Just re-run the failed ones again and if they pass => it's all good!_

_Put an `x` in the boxes that apply._

## Pull Request Type
Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [x] Code style update (formatting, new lines, etc.)
- [ ] Refactoring (no functional changes, renaming)
- [x] Small improvement (fix typo, UI fine-tune, change color or something small)
- [ ] Gradle Build related changes
- [ ] Dependencies update (updating libraries)
- [ ] Documentation (clarifying comments, KDoc)
- [ ] Tests (Unit, Integration, UI tests)
- [ ] Other (please describe):

_Put an `x` in the boxes that apply._

## Does this PR closes any GitHub Issues?
Check **[Ivy Wallet Issues](https://github.com/ILIYANGERMANOV/ivy-wallet/issues)**.
- Closes #684 (type issue number here)

## What's changed?
Describe with a few bullets **what's new:**
- Added expense description under title in 
- b
- c
- d

## How to run Ivy Wallet's UI tests (`androidTest`)
**Connect Android Emulator**
- Pixel 5 API 29+ AVD emulator _(recommended)_
- Pixel 3XL API 29+ AVD emulator _(recommended)_
- Large screen physical device _(might also work)_

**Method 1: Android Studio UI**
- Find `com (androidTest)` package
- Right click
- `Run 'Tests in 'com''`

_Note: If you've checked "Compact Middle Packages" the option will appear as `com.ivy.wallet (androidTest)`._

**Method 2: Gradle Wrapper**
- `chmod +x gradlew` (Linux)
- `./gradlew connectedDebugAndroidTest`

**Method 3: Fastlane**
- Install Ruby 2.7
- `bundle install`
- `bundle exec fastlane ui_tests`

### CODE REVIEW

**Comment by mhss1** [comment]
## This is the results:

when the description is short:


when the description is long (more than 3 lines):

File: app/src/main/java/com/ivy/wallet/ui/theme/transaction/TransactionCard.kt [hunk 1]
```diff
@@ -110,6 +111,24 @@ fun LazyItemScope.TransactionCard(
 
         }
 
+        if (transaction.description.isNotNullOrBlank()){
+            Spacer(
+                Modifier.height(
+                    if (transaction.title.isNotNullOrBlank()) 4.dp else 8.dp
```
Comment (ILIYANGERMANOV) [inline app/src/main/java/com/ivy/wallet/ui/theme/transaction/TransactionCard.kt:117]: Shouldn't it be?
`if (transaction.description.isNotNullOrBlank()) 4.dp else 8.dp`

**Review by ILIYANGERMANOV — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/ivy/wallet/ui/theme/transaction/TransactionCard.kt (hunk 1 above)
Comment (mhss1) [reply app/src/main/java/com/ivy/wallet/ui/theme/transaction/TransactionCard.kt:117]: if the description is null or blank the code will not even reach this point at all. the Spacer and the Text are added only if the description is not null or blank and then if there is a title, the space between the title and the description will be 4dp and if not the space will be 8dp which is the space between the description and the header

**Review by ILIYANGERMANOV — APPROVED** [review]

**Comment by ILIYANGERMANOV** [comment]
Got it. Thank you for your contribution! Please, let me know your LinkedIn so I can endorse you :)

**Comment by mhss1** [comment]
Thank you and I will do my best to contribute more in the future.
This is my linkedIn: https://www.linkedin.com/in/mhss/

**Pull request merged**